From 01ee8986f739057c07f6ad460de423979c208bf1 Mon Sep 17 00:00:00 2001
Message-ID: <01ee8986f739057c07f6ad460de423979c208bf1.1732384228.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1732384228.git.sharlatanus@gmail.com>
References: <cover.1732384228.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Thu, 21 Nov 2024 21:49:24 +0000
Subject: [PATCH 46/46] gnu: python-gatspy: Adjust inputs.

* gnu/packages/astronomy.scm (python-gatspy) [native-inputs]: Remove
python-nose.

Change-Id: Ifbbaedf62817339743ac2313626cd058134479d9
---
 gnu/packages/astronomy.scm | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 999914b3f9..4370f8e1c3 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2210,12 +2210,14 @@ (define-public python-gatspy
           ;; Tests need this
           (add-before 'check 'set-HOME
             (lambda _ (setenv "HOME" "/tmp"))))))
+    (native-inputs
+     (list python-pytest
+           python-setuptools-scm))
     (propagated-inputs
      (list python-astroml
            python-numpy
            python-scipy
            python-supersmoother))
-    (native-inputs (list python-pytest python-nose python-setuptools-scm))
     (home-page "https://github.com/astroml/gatspy")
     (synopsis "General tools for astronomical time series in Python")
     (description
-- 
2.46.0

